source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -146,36 +146,6 @@ namespace
         }
     }
 
-    void readUiPosition(const ElementPtr& src, PvtObject* dest)
```

Comment:
Since layout position is stored as string we can use the generic metadata mechanism now, so remove the explicit handling.